packages/@aws-cdk/aws-neptune/README.md

```diff
@@ -143,3 +143,36 @@ new neptune.DatabaseCluster(this, 'Cluster', {
   autoMinorVersionUpgrade: true,
 });
 ```
+
+## Logging
+
+Neptune supports various methods for monitoring performance and usage. One of those methods is logging
+
+1. Neptune provides logs e.g. audit logs which can be viewed or downloaded via the AWS Console. Audit logs can be enabled using the `neptune_enable_audit_log` parameter in `ClusterParameterGroup` or `ParameterGroup`
+2. Neptune provides the ability to export those logs to CloudWatch Logs
+
+```ts
+// Cluster parameter group with the neptune_enable_audit_log param set to 1
+const clusterParameterGroup = new neptune.ClusterParameterGroup(this, 'ClusterParams', {
+  description: 'Cluster parameter group',
+  parameters: {
+    neptune_enable_audit_log: '1'
```

Comment:
It would be cool if we could somehow do this automatically, but it doesn't look like we have that capability in these constructs currently. I'll consider it out of scope, but just wanted to leave a note.

Reply:
Absolutely, it'd be gr8 if that's handled by default. But to achieve that in a nice fashion, I think
1. Maybe the cluster needs to create default cluster and instance parameter groups
2. Expose methods to allow setting parameters on those groups - while also refactoring parameter groups to support exposing what parameters are there and adding/overriding the existing ones

All in all, maybe an idea for a separate PR